scripts/import/import_variant_synonyms

```diff
@@ -165,6 +167,45 @@ elsif($source_name eq 'UniProt'){
   add_synonyms_uniprot(\%synonym, $source_name, $variation_set_id, $dba, $species); 
 }
 
+elsif($source_name eq 'EVA') {
+  my $result_synonyms = parse_eva_file($data_file);
+
+  # Source EVA
+  my %source_eva = (
+  "EVA" => {
+    description => "Short variant data imported from EVA",
+    url         => "https://www.ebi.ac.uk/eva/",
+    status      => "germline",
+  }, 
+  );
+
+  # get source info 
+  $source_description ||= $source_eva{$source_name}->{description};
+  $source_url ||= $source_eva{$source_name}->{url};
+  my $source_status = $source_eva{$source_name}->{status};
+
+  my $source = get_source($species, $dba, $source_name, $source_version, $source_url, $source_description, $source_status);
```

Comment:
`get_source()` is hardcoded for human. Maybe we should replace [line 635](https://github.com/Ensembl/ensembl-variation/blob/b5c2fcc0ee634f2cb1996b1b25b0ff262bb12a02/scripts/import/import_variant_synonyms#L635) with:
```perl
my $source_adaptor = $dba->get_SourceAdaptor($species, 'variation', );
```

There are other hardcoded mentions to human in this script, maybe we should use `$species` where possible instead?

Reply:
I updated line 635.
The other mention is in the dbSNP source which is specific to human.